orc8r/cloud/go/services/state/servicers/southbound/validate.go

```diff
@@ -1,15 +1,15 @@
 /*
- * Copyright 2020 The Magma Authors.
- *
- * This source code is licensed under the BSD-style license found in the
- * LICENSE file in the root directory of this source tree.
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
+ Copyright 2020 The Magma Authors.
+
+ This source code is licensed under the BSD-style license found in the
+ LICENSE file in the root directory of this source tree.
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+*/
 
 package servicers
```

Comment:
Do the fns in this pkg need to be public? Is fine if yes, but isn't clear to me that yes

Reply:
Yeah it's used in the servicer 😕https://github.com/christinewang5/magma/blob/195f312f6005b04995ff95e09a1a0da8d0e623b7/orc8r/cloud/go/services/state/servicers/protected/servicer.go#L42-L42